haskoin-wallet/Network/Haskoin/Wallet/Transaction.hs

```diff
@@ -173,7 +174,7 @@ addrTxs (Entity ai _) addrI ListRequest{..} = do
         tidList = fOrd $ take (fromIntegral listLimit) $
             drop (fromIntegral listOffset) $ fOrd tids
 
-    -- Use a sliptSelect query here with the exact tids to speed up the
+    -- Use a splitSelect query here with the exact tids to speed up the
     -- query.
     res <- splitSelect tidList $ \tid ->
```

Comment:
I think that this query can be optimized as we don't need the aggregates. We can probably get rid of the left joins and the group by clause.